src/system_metrics/mongoose_system_metrics_collector.erl

```diff
@@ -38,12 +38,12 @@ report_getters() ->
     ].
 
 get_hosts_count() ->
-    Hosts = ejabberd_config:get_global_option(hosts),
+    Hosts = ejabberd_config:get_global_option_or_default(hosts, []),
```

Comment:
This would get only static hosts, I think we need to replace this with a number of domains and a number of host types.

Reply:
Please re-review.